.buildkite/copy_files.py

```diff
@@ -46,7 +46,7 @@ def perform_auth():
 
 def handle_docker_login(resp):
     pwd = resp.json()["docker_password"]
-    subprocess.call(
+    subprocess.check_call(
```

Comment:
@aslonnie for my own understanding, why do we want to use `check_call`?

Reply:
so that docker login failure can surface, rather than failing silently.